src/current/cockroachcloud/production-checklist.md

```diff
@@ -6,9 +6,63 @@ cloud: true
 docs_area: deploy
 ---
 
-This page provides important recommendations for CockroachDB {{ site.data.products.cloud }} production deployments.
+Before deploying CockroachDB {{ site.data.products.cloud }} in production, it is important to understand the Shared Responsibility Model that delineates the responsibilities of Cockroach Labs and the customer in managing CockroachDB {{ site.data.products.cloud }} clusters. 
 
-## Follow the SQL Best Practices
+Under the Shared Responsibility Model, Cockroach Labs is responsible for the following tasks: 
+
+- Cluster and cloud service availability and reliability
+- Maintenance and security of hardware and operating systems
+- Database and security patches 
+- Cluster backups 
```

Comment:
Is it worth differentiating between CRL-managed and. customer-managed backups?

Reply:
We do say `- (Optional) Taking customer-owned backups` in the list of customer tasks

Reply:
Yes, but I guess reading it on behalf of a novice customer, I might think that seeing Cluster backups listed under CRL responsibilities means I can't take my own backup. Then I need to hold that in my mind for a bit until I make it down to the second list. It just seemed like it might be nice to add the word "Automated" or something here.

Reply:
Gotcha. Done!